src/wrappers/nats.js

```diff
@@ -0,0 +1,204 @@
+/* eslint-disable camelcase */
+/**
+ * @fileoverview Handlers for Nats instrumentation
+ */
+
+const shimmer = require('shimmer');
+const {
+    tracer,
+    tryRequire,
+    eventInterface,
+    utils,
+} = require('epsagon');
+const traceContext = require('../trace_context.js');
+
+
+const NATS_TYPES = {
```

Comment:
Some of it was already defined in epsagon-node, so keep it in there and import it to here